### DISCUSSION TITLE
Join lines without vim mode

### DISCUSSION THREAD

**Discussion by mattly** [zed-industries/zed#6651] [Feature Requests]
I'd like to have a command to join the next line onto the end of the current one, akin to vim's `J` command - does this exist without using vim mode?

**Comment by iamnbutler** [comment]
Try `control + J`! I think this is the same as the VSCode binding.

**Reply by krstp** [reply]
This is gold. I've been looking for this... there are some workarounds, but this def makes work smoother. Thanks!
